Sources/_OpenAPIGeneratorCore/Translator/Content/ContentType.swift

```diff
@@ -214,12 +224,17 @@ struct ContentType: Hashable {
     /// A Boolean value that indicates whether the content type
     /// is a multipart form.
     var isMultipart: Bool { category == .multipart }
+    /// A Boolean value that indicates whether the content type
+    /// is a type of XML.
+    var isXml: Bool { category == .xml }
```

Comment:
```suggestion
    var isXML: Bool { category == .xml }
```

Nit: casing, to be consistent with JSON.